src/components/_components.scss

```diff
@@ -11,3 +11,4 @@
 @import 'WebexMemberRoster/WebexMemberRoster';
 @import 'WebexRemoteMedia/WebexRemoteMedia';
 @import 'WebexSettings/WebexSettings';
+@import 'generic/generic';
```

Comment:
use `components/generic/index.js` instead, so you can import it as `generic`

similar to `components/index.js` (which is not `components/components.js`).